docs/repo/property-pages/visibility-conditions.md

```diff
@@ -10,8 +10,9 @@ Visibility conditions are not interpreted by the back-end, which treats them as
 
 ## Specification
 
-There are three kinds of visibility conditions:
+There are three four of visibility conditions:
 - VisibilityCondition is a visibility condition on a property (whether the property should be shown).
+- EditabilityCondition is a visibility condition on a property that determines whether the property should editable (if true or not specified) or readonly (if evaluated to false).
```

Comment:
I wouldn't call this a visibility condition per se, as it doesn't affect the visibility of anything. :-)

Reply:
...should _be_ editable...